cdap-ui/app/styles/variables.less

```diff
@@ -39,6 +39,11 @@
 @brand-warning:         rgb(244, 180, 0);   // #f4b400
 @brand-danger:          rgb(209, 86, 104);  // #d15668
 
+@cdap-light-grey:       #999999;
+@cdap-light-green:      #00C853; 
+@cdap-blue:             #4761e7;
+@cdap-light-red:        #FF6600;
```

Comment:
Can we please replace the usage of #FF6600 for the FAILED status, to use #D15668. We want to use the Red, not Orange. 

CC @leacd